### PR TITLE
Feat/tvl tokens

### PR DESCRIPTION
**Pull request by hochbruj**
https://trello.com/c/klHS1Fl7

Additions to that task:
- Added 24h volume to token table
- Filtered out tokens without usd amount from token list
- Changed currency format to 2 decimals for tvl and volume amounts (price with 4 decimals unchanged)

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -84,19 +84,43 @@ export default function Home() {
...
+          ...acc,
+          [token0]: {
+            ...curr.token0,
+            tvl: acc[token0] ? acc[token0].tvl + reserve0Usd : reserve0Usd,
+            volume: acc[token0] ? acc[token0].volume + volume0 : volume0,
+          },
+          [token1]: {
+            ...curr.token1,
+            tvl: acc[token1] ? acc[token1].tvl + reserve1Usd : reserve1Usd,
+            volume: acc[token1] ? acc[token1].volume + volume1 : volume1,
```
Comment (OliverNChalk) [inline pages/index.js]:
```suggestion
          ...acc,
          [token0]: {
            ...curr.token0,
            tvl: (acc[token0]?.tvl ?? 0) + reserve0Usd,
            volume: (acc[token0]?.volume ?? 0) + volume0,
          },
          [token1]: {
            ...curr.token1,
            tvl: (acc[token1]?.tvl ?? 0) + reserve1Usd,
            volume: (acc[token1]?.volume ?? 0) + volume1,
```

File: pages/index.js [hunk 2]
```diff
@@ -84,19 +84,43 @@ export default function Home() {
...
       setPairs(_pairs);
+
+      //transform to tokens
+      const tokensMap = _pairs.reduce((acc, curr) => {
+        const token0 = curr.token0.contractAddress;
+        const token1 = curr.token1.contractAddress;
+        const reserve0Usd = +curr.token0Reserve * curr.token0.usdPrice;
+        const reserve1Usd = +curr.token1Reserve * curr.token1.usdPrice;
+        const volume0 = +curr.token0Volume * curr.token0.usdPrice;
+        const volume1 = +curr.token1Volume * curr.token1.usdPrice;
```
Comment (OliverNChalk) [inline pages/index.js:99]: New line before return is nice for readability

**Comment by xenide** [comment]
image

In landscape mode on mobile, the justification seems to depend on the length of the name. Should we standardize it to left justify?

**Review by KennethAshley — APPROVED** [review]
Oliver has commented on the issues I'd bring up. Once fixed you have my approval.

**Comment by hochbruj** [comment]
Implemented Oli's review comments.

File: pages/index.js [hunk 3]
```diff
@@ -84,19 +84,44 @@ export default function Home() {
...
+  //   const getTokens = async () => {
+  //     const tokensApiResult = await axios(`${API_BASE_URL}tokens`);
+  //     const _tokens = Object.values(tokensApiResult.data);
+  //     setTokens(_tokens);
+  //   };
 
-    getTokens();
-  }, []);
+  //   getTokens();
+  // }, []);
```
Comment (OliverNChalk) [inline pages/index.js]: Do we want to leave old code as a comment, does it need to be brought back in the future or why is it here?

**Review by OliverNChalk — APPROVED** [review]

File: pages/index.js (hunk 3 above)
Comment (hochbruj) [reply pages/index.js]: I removed it

**Pull request merged**